assembly/assembly-machine-war/src/main/webapp/WEB-INF/classes/codenvy/che-machine-configuration.properties

```diff
@@ -27,8 +27,8 @@ vfs.local.fs_index_root_dir=${catalina.base}/temp/indexes
 vfs.local.tmp_workspace_fs_root_dir=${catalina.base}/temp/tmp-ws-fs-root
 vfs.local.directory_mapping_file=${catalina.base}/temp/vfs
 
-#this path is relative to user home directory
-che.workspace.metadata = che/.workspace
+che.workspace.metadata = .metadata/plugins/org.eclipse.jdt.core
```

Comment:
Are you sure that it is che.worskpace.metadata folder? Maybe che.worskpace.jdt.metadata